.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,42 @@
+# A CI configuration to check for licenses in Dart files.
+
+name: Changelog Check
+
+# Callers of this workflow should use it as follows:
+#
+# name: License Check
+# on:
+#   pull_request:
+#     branches: [ main ]
+#     types: [opened, synchronize, reopened, labeled, unlabeled]
+#   push:
```

Comment:
We can probably skip the push / tags section - that was for the publish workflow which needed to react to commit tags on main.